build.sh

```diff
@@ -120,7 +147,12 @@ main() {
       clean
       ;;
     *)
-      build_cre2
+      if [[ "$OS" == "Darwin" ]]; then
+        build_cre2 x64
```

Comment:
for CMAKE we want these to be `arm64` and `x86_64`.
For .NET the RIDs are `osx-arm64` and `osx-x64`.

Reply:
Hopefully fixed now